ingest-mixed-generator/main.py

```diff
@@ -69,18 +81,29 @@ def main(**kwargs):
 
     producer = get_kafka_producer(settings)
 
+    print("Generating attachment data...", flush=True)
+
+    payloads = generate_attachment_payloads(settings)
```

Comment:
Let's either:
1. Move these inside `send_messages()`, and only print the message when we actually have something to generate.
2. Have two major branches (depending on whether `num_attachments` is non-zero): one will be `send_messages()` (without modifications, the current default behavior), and another something like `send_attachments()`, where you do your own custom logic. Hopefully this will make it more explicit that the two modes are somewhat different/incompatible.